src/pytest_mock_resources/fixture/moto/action.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+import io
+from dataclasses import dataclass
+from typing import BinaryIO, ClassVar, Iterable, TextIO, TYPE_CHECKING, Union
+
+from pytest_mock_resources.action import AbstractAction, validate_actions
+
+if TYPE_CHECKING:
+    from pytest_mock_resources.fixture.moto.base import Session
+
+
+ObjectContent = Union[bytes, str, BinaryIO, TextIO]
+
+
+@dataclass
+class S3Bucket(AbstractAction):
```

Comment:
I think our namespacing is the issue - since these are part of the overall public facing api. They should be named so that they make sense in the grand scheme of things.

Reply:
I can't tell if this means you think they're fine as-is or that they need to change. As-is, i think they make sense in the grand scheme of things.

Even in the unlikely world that something came along and replaced moto, S3Bucket would be a correct name that could be used for either fixture.

Reply:
fine as-is